docs/user/openstack/provider_networks.md

```diff
@@ -0,0 +1,115 @@
+# Provider Networks
+
+
+## Table of Contents
+- [Provider Networks](#provider-networks)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Common prerequisites](#common-prerequisites)
+  - [Considerations when creating provider networks](#considerations-when-creating-provider-networks)
+  - [Deploying cluster with primary interface on a provider network with IPI](#deploying-cluster-with-primary-interface-on-a-provider-network-with-ipi)
+  - [Known issues](#known-issues)
+
+
+## Introduction
+
+Provider networks map directly to an existing physical network in a data center.
+Example of network types include flat (untagged), VLAN (802.1Q tagged) and VXLAN. 
+OpenShift clusters that are installed on provider networks do not require tenant networks or floating IP addresses (FIPs).
+Therefore, the OpenShift installer does not create these resources during installation.
+More information can be found about provider networks terminology [here][1].
+
+Here is a basic architecture of one OCP cluster running on a provider network and another one
+on a tenant network:
+
+![OCP on a provider network](provider-network.png)
+
+
+## Prerequisites
+
+* The [Neutron service][2] is enabled and accessible through the [OpenStack Networking API][3].
+* The Neutron service is configured with the [port-security and allowed-address-pairs][4] extensions so the installer can
+  add the `allowed_address_pairs` attribute to ports.
+
+
+## Considerations when creating provider networks
+
+* The provider network has to be shared with other tenants, otherwise Nova won't be able to request ports on that external
+  network. For more details, see [BZ#1933047][5].
+
+        openstack network create --share (...)
+
+  To secure that network, it is advised to create [RBAC][6] rules so the network can be only usable by a specific project.
+
+
+* The project that you use to install OpenShift must own the provider network.
+
+    The provider network and the subnet must be owned by the project that is used to install OpenShift instead of `admin`.
+    If they are not, Terraform fails while creating ports. 
```

Comment:
If they run the installer from the admin user, this should work still, right? It's not the user experience we want, but maybe we should inform the user that its a permission issue and not impossible: "If the are not, you will have to run the installer from the admin user to create ports on the network."